### PR TITLE
Update my affiliation for Tomoya Usami

### PR DESCRIPTION
**Pull request by hiyosi**

### CODE REVIEW

File: developers_affiliations2.txt [hunk 1]
```diff
@@ -10101,8 +10101,8 @@ hitmoon: hitmoon!users.noreply.github.com, zxq_yx_007!163.com
 	NetEase
 hiyokotaisa: hiyokotaisa!users.noreply.github.com
 	Red Hat
-hiyosi: hiyosi!users.noreply.github.com, tousami!zlab.co.jp
-	zlabjp
+Tomoya Usami: usami.tomoya!gmail.com, hiyosi!users.noreply.github.com, tousami!zlab.co.jp
```
Comment (lukaszgryglicki) [inline developers_affiliations2.txt:10104]: This (Tomoya Usami) must a a GitHub user id, it is the same as name part for `@users.noreply.github.com`: https://github.com/hiyosi

**Comment by lukaszgryglicki** [comment]
Will fix username post-merge.

File: developers_affiliations2.txt (hunk 1 above)
Comment (hiyosi) [reply developers_affiliations2.txt:10104]: sorry, I misunderstood.

**Comment by hiyosi** [comment]
Thank you!

**Pull request merged**